src/components/Clock/Clock.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+
+interface State {
+  date: string;
+}
+
+interface Props {
+  name: string,
+}
+
+export class Clock extends React.Component <Props, State> {
+  state:Readonly<State> = {
+    date: '',
+  };
+
+  today = new Date().toUTCString().slice(-12, -4) ;
```

Comment:
do you ever use this variable?

Reply:
consider creating a function to return you this date `new Date().toUTCString().slice(-12, -4)`